src/main.tsx

```diff
@@ -1,35 +1,2 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import { CommentSection } from './CommentSection'
-import { Filters } from './CommentFilters'
-import type { CommentOptions } from './types'
-
-export const BlueskyComments = {
-  init: function(elementId: string, options: CommentOptions) {
```

Comment:
Personally I don't see a use case for this given that in the end it is a React component. 

If I wasn't using React, I'd reach for a different solution:
https://github.com/ascorbic/bluesky-comments-tag

Reply:
Hmm, that does look like a nice option, though I don't think it supports the user-based mode? Or filters, etc.

Is there any issue with leaving this option in while also getting the improvements here added? As someone with a blog that is already using the library this way I don't want to have to have to change solutions (especially since I'm not sure any other options have feature parity with this one).

Reply:
Ok, I was able to get this working so I think we can move forward on this path